functionsAndScope.js

```diff
@@ -0,0 +1,285 @@
+
+// ## Question 3
+
+// Write a function `arraySum` that takes in an array and returns the sum of all the elements.
+
+// Example:
+
+// Input: `[13,2,4,-2]`
+
+// Output: `17`
+
+function arraySum (arr) {
+    let sum = 0;
+    arr.forEach(element => {
+        sum += element;
+    });
+    return sum;
+}
+
+console.log("3) ", arraySum([1,2,3,4,5]));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+
+// ## Question 4
+
+// Write a function that takes in any two numbers and return their sum.
+
+// Example:
+
+// Input: `14, 5`
+
+// Output: `19`
+
+const addMeUp = (one, two) => one + two;
+
+console.log("4) ", addMeUp(23, 24));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+// ## Question 5
+
+// Write a function takes in any number grade and returns a corresponding letter grade.
+
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+
+
+// Example:
+
+// Input: `88`
+
+// Output: `"B"`
+
+const passOrFail = (grade) => {
+    return grade === 100 ? "A"
+        : grade > 89 ? "A"
+        : grade > 79 ? "B"
+        : grade > 69 ? "C"
+        : grade > 64 ? "D"
+        : "F";
+}
+
+console.log("5) ", passOrFail(88));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+// ## Question 6
+
+// Implement a function named `repeatLog` that takes a string and a number as parameters. The function should log `message` `count` number of times.
+
+// Example:
+
+// Input: `"+", 10`
+
+// Output: `++++++++++`
+
+const repeatLog = (string, number) => {
+    let printMe = '';
+    for (let i = 0; i < number; i++) {
+        printMe += string;
+    }
+    return printMe;
+}
+
+console.log("6) ", repeatLog("+", 10));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+// ## Question 7
+
+// Write a function named `first` that takes a number and returns an array with all the numbers from 1 to n
+
+
+// Example:
+
+// Input: `first(3)`
+
+// Output: `[1, 2, 3]`
+
+const first = (number) => {
+    let arr = [];
+    for (let i = 1; i <= number; i++) {
+        arr.push(i);
+    }
+    return arr;
+}
+
+console.log("7) ", first(3));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+// ## Question 8
+
+// Write a function that logs the numbers from 1 to x, except:
+
+// If the number if a multiple of 3, log `"Fizz"` instead of the number
+// If the number is a multiple of 5, log `"Buzz"` instead of the number
+// If the number is a multiple of 3 AND 5, log `"FizzBuzz"` instead of the number
+// Your function should take in one parameter: x (the number to count up to)
+
+const fizzBuzzMe = (number) => {
+    return number % 3 === 0 && number % 5 === 0 ? "FizzBuzz"
+        : number % 3 === 0 ? "Fizz" 
+        : number % 5 === 0 ? "Buzz"
+        : "";
+}
+
+console.log("8) ", fizzBuzzMe(15));
```

Comment:
Because you have a return at the top, it stops. what can we do to fix this?
Lastly, it seems that we aren't getting all of the numbers from 1 until that number .